fixtures/datagen/fakerize.py

```diff
@@ -379,11 +379,12 @@ def store(fakes, kind="vasps", directory=OUTPUT_DIRECTORY):
     Each file should be the name of the fakerized uuid
     Return the path
     """
+    directory = os.path.join(directory, kind)
     if not os.path.exists(directory):
         os.makedirs(directory)
 
     for idx, fake in fakes.items():
-        fname = os.path.join(directory, kind + "::" + shorten(idx) + ".json")
+        fname = os.path.join(directory, shorten(idx) + ".json")
         with open(fname, "w") as outfile:
```

Comment:
Instead of storing the files in `vasps:1234.json` and `certreqs::4568.json` we're using a directory to split the different records up, e.g. `vasps/1234.json` and `certreqs/4568.json`.